### PR TITLE
[TDEngine] Remove REST support

### PR DESCRIPTION
**Pull request by Eyal-Danieli**
Install `taospy[ws]` that includes python connector to TDEngine using either REST API or WebSocket also includes installation of unnecessary dependencies such as  `pytest-cov`.
Therefore, at the moment we are going to support only the WebSocket connection that can be implemented using `taos-ws-py` which doesn't include the unnecessary dependencies.

### CODE REVIEW

File: storey/targets.py [hunk 1]
```diff
@@ -885,33 +877,17 @@ def __init__(
...
         _Batching._init(self)
         _Writer._init(self)
-
-        parsed_url = urlparse(self._url)
-
-        if parsed_url.scheme == "taosws":
-            import taosws
-
-            self._using_websocket = True
+        if self._url.startswith("taosws://"):
```
Comment (assaf758) [inline storey/targets.py:887]: this will be true whether u/p was provided or not, right? if that's the case need to improve the logic.

File: storey/targets.py [hunk 2]
```diff
@@ -819,13 +816,9 @@ def __init__(
         table_col: Optional[str] = None,
         supertable: Optional[str] = None,
         tag_cols: Union[str, List[str], None] = None,
-        timeout: Optional[int] = None,
         time_format: Optional[str] = None,
         **kwargs,
     ):
```
Comment (assaf758) [inline storey/targets.py:822]: why did you remove the check that it's taosws?

Comment (Eyal-Danieli) [reply storey/targets.py:822]: it's only relevant to REST connection, there is no timeout param with the websocket connection

**Review by assaf758 — APPROVED** [review]
